tests/functional/ancestral/cram/infer-amino-acid-sequences-genes-file.t

```diff
@@ -0,0 +1,22 @@
+Setup
+
+  $ source "$TESTDIR"/_setup.sh
+
+Infer ancestral nucleotide and amino acid sequences, using a genes file.
+
+  $ ${AUGUR} ancestral \
+  >  --tree $TESTDIR/../data/tree.nwk \
+  >  --alignment $TESTDIR/../data/aligned.fasta \
+  >  --annotation $TESTDIR/../data/zika_outgroup.gb \
+  >  --genes $TESTDIR/../data/genes.txt \
+  >  --translations $TESTDIR/../data/aa_sequences_%GENE.fasta \
+  >  --output-node-data "$CRAMTMP/$TESTFILE/ancestral_mutations.json" \
```

Comment:
Wait - didn't we [just discuss](https://github.com/nextstrain/augur/pull/1344#discussion_r1416467185) moving away from this filename structure to a flat structure (`--output-node-data ancestral_mutations.json`)?

Reply:
Oops, this was from copy-pasting old practices 🤦

Reply:
Updated in [force-push](https://github.com/nextstrain/augur/compare/f59f1808e9849eb8b0cab1e20d4654cf91eced4c..a60f1a63f7140e738d4e29ae67bd9c72f4c88268).